### PR TITLE
Mapbox tab for new add-custom-basemap modal

### PR DESCRIPTION
**Pull request by viddo**
screen shot 2015-05-27 at 16 02 31

I extracted the old code that was not related to the old view logic, so I could reuse it for the new modal, including test cases. Apart from that there's nothing special, just a new view/model/template for the new tab content really.

@xavijam please review
cc @saleiva

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3936/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3940/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3943/))

File: lib/assets/javascripts/cartodb/table/basemap/basemap_dialog/mapbox_basemap_pane.js [hunk 1]
```diff
@@ -301,7 +126,9 @@
       this._hideLoader();
 
       // Show error
-      this._showError(message);
+      this.$(".info p").html(message);
+      this.$el.find("input").addClass("error");
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/table/basemap/basemap_dialog/mapbox_basemap_pane.js]: this.$('input')...

**Comment by xavijam** [comment]
Looks good to me, tested in staging?

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3955/))

**Comment by viddo** [comment]
Yes, deployed on ded06 if you want to give it a try

**Pull request merged**